src/v2/spec.yaml

```diff
@@ -6806,6 +6835,45 @@ paths:
           $ref: "#/components/responses/400Response"
         "500":
           $ref: "#/components/responses/500Response"
+  /farcaster/frame/notification_tokens:
+    get:
+      tags:
+        - Frame
+      summary: |
+        List of frame notification tokens.
+      description: |
+        Returns a list of notifications tokens related for an app
+      externalDocs:
+        url: https://docs.neynar.com/reference/list-notification-tokens
+      operationId: list-notification-tokens
```

Comment:
Please update the operationId to `fetch-notification-tokens`
We use `fetch` prefix to get a list of entities

Reply:
All clear. Got confused with comments that referenced `https://docs.neynar.com/reference/list-all-channels)`
Now I see it was non existent